.travis.yml

```diff
@@ -25,9 +22,9 @@ matrix:
     - rvm: 2.1.10
       gemfile: Gemfile
     - rvm: 2.4.0
```

Comment:
Since you removed these `Gemfile`s, I think you can remove this exclude clauses altogether
```diff
- - rvm: 2.4.0
-    gemfile: gemfiles/Gemfile.rails-4-1-stable # activesupport 4.1 depends on json 1.8.3 which does not compile with Ruby 2.4.0
- - rvm: ruby-head
-    gemfile: gemfiles/Gemfile.rails-4-1-stable # activesupport 4.1 depends on json 1.8.3 which does not compile with Ruby 2.4.0